### PR TITLE
refactor: Cleaning up simple, top-level items

### PR DESCRIPTION
**Pull request by rschristian**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "preact-realworld-example-app",
-  "private": "true",
-  "homepage": "https://github.com/mutoe/preact-realworld-example-app#readme",
+  "version": "0.1.2",
+  "private": true,
```
Comment (rschristian) [inline package.json:4]: I'd say there's still value in versioning a web app, but meta data like the home page? Not really sure about when that would be useful?

Comment (mutoe) [reply package.json:4]: If a user wants to view its online repository after opening the source code, this will help him, I think

Comment (rschristian) [reply package.json:4]: If the user closes the source then they'll already know of the upstream?

Comment (mutoe) [reply package.json:4]: It doesn’t matter, I don’t mind these ;p

**Review by mutoe — APPROVED** [review]

**Pull request merged**